app/controllers/Application.scala

```diff
@@ -99,16 +99,44 @@ class Application(
     )).withSettingsSurrogateKey
   }
 
-  def contributionsLanding(countryCode: String): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
+  def contributionsLanding(countryCode: String, maybeSSR: Option[String]): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
     type Attempt[A] = EitherT[Future, String, A]
     implicit val settings: AllSettings = settingsProvider.getAllSettings()
     request.user.traverse[Attempt, IdUser](identityService.getUser(_)).fold(
-      _ => Ok(contributionsHtml(countryCode, None)),
-      user => Ok(contributionsHtml(countryCode, user))
+      _ => Ok(contributionsHtml(countryCode, None, maybeSSR.contains("on"))),
+      user => Ok(contributionsHtml(countryCode, user, maybeSSR.contains("on")))
     ).map(_.withSettingsSurrogateKey)
   }
 
-  private def contributionsHtml(countryCode: String, idUser: Option[IdUser])(implicit request: RequestHeader, settings: AllSettings) = {
+  private def contributionsHtml(countryCode: String, idUser: Option[IdUser], isTest: Boolean)(implicit request: RequestHeader, settings: AllSettings) = {
+    if (isTest) {
+      val annualAmounts: Map[String, (String, String, String, String)] = Map(
+        "uk" -> ("£50", "£100", "£250", "£500"),
+        "us" -> ("$50", "$100", "$250", "$500"),
+        "au" -> ("$100", "$250", "$500", "$750"),
+        "eu" -> ("€50", "€100", "€250", "€500"),
+        "int" -> ("£50", "£100", "£250", "£500"),
+        "nz" -> ("$50", "$100", "$250", "$500"),
+        "ca" -> ("$50", "$100", "$250", "$500")
```

Comment:
we have all the amounts correctly inserted but the second is selected by default in the placeholder.  in a AU/NZ the first one is selected by default in the real rendered copy, so the reader will see the selection end up different to the placeholder.  I don't think this is an issue but I wanted to flag it just in case.

Reply:
if we want to do amounts testing we would have to do something smart here.  Since the amounts are hard coded, we don't want to give a worse experience to the people in both tests.  We could spend a bit of time once the test is running to implement a rudimentary server side ab testing allocation to enable the amounts test.